Create 2025-07-30-Scam-domain-category-introduced.mdx (#23954)
cloudflare/cloudflare-docs: src/content/changelog/gateway/2025-07-28-Spam-domain-category-introduced.mdx

---
title: Spam domain category introduced under Security Threats

> description: You can now create policies specifically targetting the Spam content

Reviewer: ```suggestion
description: You can now create policies specifically targeting the Scam content
```

date: 2025-07-28
---
 

> We have introduced a new Security Threat category: Scam is now available and relevant domains are marked with the Scam category. Scam typically refers to fraudulent websites and schemes designed to trick victims into giving away money or personal information.

Reviewer: ```suggestion
We have introduced a new Security Threat category called **Scam**. Relevant domains are marked with the Scam category. Scam typically refers to fraudulent websites and schemes designed to trick victims into giving away money or personal information.
```

> New category added

Reviewer: ```suggestion
**New category added**
```

> Parent ID	Parent Name	Category ID	Category Name

Reviewer: ```suggestion
| Parent ID	| Parent Name	| Category ID	| Category Name |
```

> 21	Security Threats	191	Scam

Reviewer: ```suggestion
| 21  |	 Security Threats	| 191	| Scam |
```
